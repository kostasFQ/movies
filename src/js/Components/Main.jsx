import React from 'react'

const arr = [1,2,3,4,5];
const Main = () => {
  return (
    <div className="content">
      <h1>KST</h1>
      {arr.map( (item)=> {
        return <div key={item}>{item}</div>
      } )}
    </div>
  )
}

export default Main