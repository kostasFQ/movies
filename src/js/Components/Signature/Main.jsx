import React from 'react';
import PropTypes from 'prop-types';
import styles from './style.scss';


const Main = (props) => {
  const {name} = props;
  return (
    <div className="content">
      <h2 className={styles.name}>{name}</h2>
    </div>
  )
}

Main.propTypes = {
  name: PropTypes.string.isRequired
};
export default Main