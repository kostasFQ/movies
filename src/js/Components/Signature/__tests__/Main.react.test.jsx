import React from 'react';
import Main from '../Main';
import renderer from 'react-test-renderer';

test('test component', () => {
  const render = renderer.create(
    <Main name='KST'/>
  );

  const component = render.toJSON();
  console.log(render.toJSON().children);

  expect(component).toMatchSnapshot();
  expect(component.type).toBe('div');
  expect.objectContaining({
    className: 'content'
  })
  expect.objectContaining({
    children: expect.any(Object)
  })
})
