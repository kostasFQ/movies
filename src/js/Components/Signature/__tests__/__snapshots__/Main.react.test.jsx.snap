// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component 1`] = `
<div
  className="content"
>
  <h2>
    KST
  </h2>
</div>
`;
